apps/api/src/app/events/usecases/send-message/send-message.usecase.ts

```diff
@@ -215,4 +222,33 @@ export class SendMessage {
 
     return !channels.find((channel) => channel === job.type);
   }
+
+  private sumFilters(
```

Comment:
Would be great to have a unit test on this one, while doing so it might be required to extract this to a public function on a filters service perhaps? and than can be unitested for all variations